src/main.ts

```diff
@@ -264,7 +264,20 @@ const keybinds: Keybind[] = [
                 return;
             }
             const currentSwitcher: number = await getStorageValue('currentswitcher');
-            if (urlParams['page'] === 'un' && urlParams['template-overall'] === 'none')
+
+            const startPage = getStorageValue('prepDossClear') ? 'dossier' : 'un';
```

Comment:
This needs an `await`, otherwise it seems to clear the dossier regardless of what the setting is.
```suggestion
            const startPage = await getStorageValue('prepDossClear') ? 'dossier' : 'un';
```